.travis.yml

```diff
@@ -82,7 +82,6 @@ matrix:
           -multifile;\
           -network;\
           -ni;\
-          -noresolver;\
```

Comment:
Why the change?

Reply:
It is required for `specload` and therefore needed for some of the codegen tests.

Reply:
Then maybe make it mandatory? (src/plugins/CMakeLists.txt)

Reply:
It is not actually mandatory, since `specload` isn't mandatory. Also `specload` can work with other resolvers, it is just that right now `noresolver` is the only one that works. Other resolvers don't call the storage plugin, if no storage file exists.